src/test/java/seedu/address/storage/JsonAdaptedPersonTest.java

```diff
@@ -51,6 +51,8 @@ public class JsonAdaptedPersonTest {
     private static final List<JsonAdaptedAppointment> VALID_APPOINTMENTS = BENSON.getAppointments().stream()
                                                                                  .map(JsonAdaptedAppointment::new)
                                                                                  .collect(Collectors.toList());
+    private static final String VALID_MEDCON = BENSON.getMedCon().toString();
```

Comment:
It would be good to create some test cases in this test class for your new Command